### DISCUSSION TITLE
Add `Bottom` type to `DefaultCartesianMarker.LabelPosition`

### DISCUSSION THREAD

**Discussion by L-Andrade** [patrykandpatrick/vico#701] [Questions] [answered]
Hello,

We've recently integrated Vico into our app to show line charts and it was fairly straightforward to add it. So far, it looks really nice and we thank you for the library.

I have just one suggestion - would it be possible to add a `Bottom` type to the LabelPosition enum? We have a marker that we display at the top of the graph (`Top`), and then we have one that is supposed to be displayed at the bottom. But currently we are using `AroundPoint`, since `Bottom` is not supported. This causes the marker's text to sometimes be on top of the chart line. Here's an example:
<img width="341" alt="Screenshot 2024-05-14 at 14 29 46" src="https://github.com/patrykandpatrick/vico/assets/20290474/075ae058-5441-4d17-9ff1-08f702c6b10a">

Is this something that can be done as suggested, or did we miss some other way to have this behaviour?

**Comment by L-Andrade** [comment, marked as answer]
Solved with https://github.com/patrykandpatrick/vico/pull/721

**Reply by patrickmichalik** [reply]
Thanks for contributing! The changes from your PR are included in [Vico 2.0.0 Alpha 21](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.21).
